Source/RpmAvatarCreator/Public/NARF/Elements/RpmColorSelectionWidget.h

```diff
@@ -0,0 +1,93 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Blueprint/UserWidget.h"
+#include "NARF/Buttons/RpmAssetButton.h"
+#include "RpmAvatarCreatorTypes.h"
+#include "RpmColorSelectionWidget.generated.h"
+
+USTRUCT(BlueprintType)
+struct FRpmColorAsset
```

Comment:
I disagree with this structure. It's making the code more complex, and it has no reason to be introduced. This is the same thing as introducing the RPMInteger that holds an integer.